docs/pico_sdk.md

```diff
@@ -63,21 +66,28 @@ In order to use the RF24 libraries in your RP2040 based project:
 
    Alternatively you can add the RF24* repositories as [git submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules).
 2. Include their CMakeLists.txt files from the RF24 libraries in your project's top-level
-   CMakeLists.txt file (usually located in the "src" directory). The following snippet
-   assumes that your project's "src" directory is on the same level as the previously
-   mentioned "lib" directory.
+   CMakeLists.txt file (usually located in the `src` directory). The following snippet
+   assumes that your project's `src` directory is on the same level as the previously
+   mentioned `lib` directory.
    ```txt
    include(../lib/RF24/CMakeLists.txt)
    include(../lib/RF24Network/CMakeLists.txt)
    include(../lib/RF24Mesh/CMakeLists.txt)
    ```
 3. In the same CMakeLists.txt file from step 2, add the RF24 libraries into the
-   `target_link_libraries` configuration.
+   `target_link_libraries` configuration:
    ```txt
-   target_include_directories(target_name PRIVATE ${CMAKE_CURRENT_LIST_DIR})
+   target_link_libraries(${CMAKE_PROJECT_NAME}
+   ... Your project's other libraries ...
```

Comment:
This should be written like an inline comment 
```text
# ... Your project's other libraries here...
```

Reply:
Sure, no problem. Changed.